aptos-move/aptos-gas-schedule/src/gas_schedule/move_stdlib.rs

```diff
@@ -31,5 +31,8 @@ crate::gas_schedule::macros::define_gas_parameters!(
         [string_index_of_base: InternalGas, "string.index_of.base", 1470],
         [string_index_of_per_byte_pattern: InternalGasPerByte, "string.index_of.per_byte_pattern", 73],
         [string_index_of_per_byte_searched: InternalGasPerByte, "string.index_of.per_byte_searched", 36],
+
+        [bcs_serialized_size_per_byte_serialized: InternalGasPerByte, { RELEASE_V1_18.. => "bcs.serialized_size.per_byte_serialized" }, 36],
```

Comment:
I'd recommend you add a constant base cost to make the model more robust.

Reply:
done